contracts/lib/Organization.sol

```diff
@@ -235,15 +279,25 @@ contract Organization is OrganizationInterface, IsMemberInterface {
     }
 
     /**
-     * @notice Checks if an address is currently registered as the owner.
+     * @notice Checks if an address is currently registered as the organization.
+     *
+     * @dev It is an implementation detail of this contract that the admin can
+     *      act on behalf of the organization. To the outside, an "admin"
+     *      doesn't exist. See also the `admin` storage variable.
      *
-     * @param _owner Address to check.
+     * @param _organization Address to check.
      *
-     * @return isOwner_ True if the given address is the owner of the
-     *                  organization. Returns false otherwise.
+     * @return isOrganization_ True if the given address represents the
+     *                         organization. Returns false otherwise.
      */
-    function isOwner(address _owner) external view returns (bool isOwner_) {
-        isOwner_ = _owner == owner;
+    function isOrganization(
+        address _organization
+    )
+        external
+        view
+        returns (bool isOrganization_)
+    {
+        isOrganization_ = _organization == owner || _organization == admin;
     }
 
```

Comment:
A proposal to rename the function to isActiveWorker as that's what we are checking, however, we could also assume that isWorker is a better interface for isActiveWorker. Thoughts?

Reply:
I think `isWorker` is a better interface. It is an implementation detail of this organization that workers can expire. Please resolve the comment if you agree.

Reply:
If `isWorker` would be part of interface, then yes, being active or expired is an implementation detail. However, expiration height is part of this contract interface (being part of its events and public functions), hence it's not an implementation detail. I am more inclined for renaming it to isActiveWorker, however, if you think isWorker is simpler and from documentation it's clear that it's about being registered and not expired, let's keep it.